frontend/src/Frontend/KadenaAddress.hs

```diff
@@ -101,21 +102,31 @@ data KadenaAddressError
   | InvalidHumanReadablePiece Text
   deriving (Show, Eq)
 
+data KadenaAddressPrefix
+  = KadenaAddressPrefix_Keep
+  | KadenaAddressPrefix_None
+  deriving (Show, Eq)
+
+data KadenaAddress_AccountCreated
```

Comment:
Don't think we actually use `_` outside of field/constructor prefixes, convenient as it would be. Might mess with lens-making?